features/step_definitions/auction_status_steps.rb

```diff
@@ -53,8 +53,10 @@
   )
 end
 
-Then(/^I should see the time I placed my bid$/) do
+Then(/^I should see a status confirmation for my sealed bid$/) do
```

Comment:
the format we are using in the rest of this file is a bit inconsistent but mostly this format:

> I should see the unpublished auction message for guests

I think we are not calling it 'status confirmation' anywhere else. status box? maybe

> I should see a status message that confirms I placed a bid

?